bin/mark_changepoints_in_json

```diff
@@ -65,13 +65,15 @@ class Segments(object):
     """A list of Segments for a whole run sequence.
     """
 
-    def __init__(self, delta, length, steady_state, cpts, means, variances, data, outliers):
+    def __init__(self, delta, half_bound, steady_state, length, cpts, means,
+                 variances, data, outliers):
         self.delta = delta
+        self.half_bound = half_bound
```

Comment:
Sort of, but it is a little confusing. I don't know what a better name would be though :/